content/en/flux/migration/_index.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: "Migration"
-linkTitle: "Migration"
+title: "Migrate from Flux v1"
+linkTitle: "Migrate from Flux v1"
 description: "Migration guides for Flux v1 and Helm Operator users."
 weight: 60
```

Comment:
Can you please also set the weight to 150

Reply:
@stefanprodan done